docs/CSBR.md

```diff
@@ -1749,23 +2357,76 @@ f. __Certificate Field:__ `subject:countryName` (OID: 2.5.4.6)
+
+Effective as of 15 September 2024, the CA SHALL ensure that, at time of issuance, the values within these fields have been disclosed within the latest publicly-available disclosure, as described in [Section 3.2.10](#3210-disclosure-of-verification-sources), as acceptable values for the applicable Incorporating Agency or Registration Agency.
 
 d. __Certificate Field:__ `subject:serialNumber` (2.5.4.5)  
    __Required/Optional:__ Required  
-   __Contents__: As specified in Section 9.2.5 of the EV Guidelines.
+   __Contents__: 
+
+* For Private Organizations, this field MUST contain the Registration (or similar) Number assigned to the Subject by the Incorporating or Registration Agency in its Jurisdiction of Incorporation or Registration, as appropriate.  If the Jurisdiction of Incorporation or Registration does not provide a Registration Number, then the date of Incorporation or Registration SHALL be entered into this field in any one of the common date formats.
+* For Government Entities that do not have a Registration Number or readily verifiable date of creation, the CA SHALL enter appropriate language to indicate that the Subject is a Government Entity.
+* For Business Entities, the Registration Number that was received by the Business Entity upon government registration SHALL be entered in this field.  For those Business Entities that register with an Incorporating Agency or Registration Agency in a jurisdiction that does not issue numbers pursuant to government registration, the date of the registration SHALL be entered into this field in any one of the common date formats.
+
+Effective as of 15 September 2024, if the CA has disclosed a set of acceptable format or formats for Registration Numbers for the applicable Registration Agency or Incorporating Agency, as described in [Section 3.2.10](#3210-disclosure-of-verification-sources), the CA MUST ensure, prior to issuance, that the Registration Number is valid according to at least one currently disclosed format for that applicable Registration Agency or Incorporating agency.
 
 e. __Certificate Field:__ Subject Physical Address of Place of Business Fields  
-   __Required/Optional:__ Required  
-   __Contents__: As specified in Section 9.2.6 of the EV Guidelines.
+   __Required/Optional:__ As stated in [Section 7.1.4.2](#71423-subject-distinguished-name-field---non-ev-code-signing-certificates) b, c, d, e and f.  
+   __Contents__: This field MUST contain the address of the physical location of the Subject's Place of Business.
+
+f. __Certificate Field__: `subject:organizationIdentifier` (OID: 2.5.4.97)  

```

Comment:
The addition of this text could be construed as a new requirement, as the CSBRs previously had no restriction on the contents of this attribute.

Not sure if any CA would actually be affected by the inclusion of this language, though.

Reply:
As agreed in yesterday's CSCWG Teleconference, the orgId language (and associated appendices) will be removed from this ballot and addressed in a future ballot, evaluating language from other WGs (S/MIME having the latest revision on the associated orgId requirements).

Reply:
Addressed in https://github.com/cabforum/code-signing/commit/46da8115bd6ceea0a49d8e5d02527777ca28a24a